id.html

```diff
@@ -0,0 +1,238 @@
+            </i>Player
+          </h4>
+        </a>
+        <p>Mode serangan ganda, untuk menghindari terdeteksi oleh server dari damage yang tinggi. Hapus cooldown, ledakan elemen tidak perlu menunggu cooldown, karena variasi permainan, server tidak akan memeriksanya. Mode tak terkalahkan probabilitas yang sudah di sesuaikan.</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-dituleiditu"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Dunia
+          </h4>
+        </a>
+        <p>Secara otomatis menyelesaikan tantangan dunia, memasak makanan secara otomatis, menghancurkan objek secara otomatis, memancing ikan secara otomatis, mengambil item secara otomatis, mengambil item dengan jangkuan yang jauh, fairy spirits secara otomatis, dialog NPC otomatis (dapat di percepat), menebang pohon secara otomatis, menyamarkan Waktu In-game, Cloing global, Kill Aura, Membuat makhluk adsorpsi, drop adsorpsi, dengan teleportasi otomatis agar tidak membuat tangan anda lelah, disesuaikan untuk hati anda.</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-24gl-paperPlane"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Teleport
+          </h4>
+        </a>
+        <p>Teleportaasi sekali klik, teleportasi sekali klik dari peta, teleportasi khusus (dengan file json untun membuka kunci dunia baru), teleportasi sepenuhnya otomatis (delay yang dapat disesuaikan), teleportasi peta (titik tempat teleportasi mana), teleportasi titik quest  (perlu melacak quest).</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-toushifenxi"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Perspektif
+          </h4>
+        </a>
+        <p>Mode Box drawing, Mode drawing Box model, Mode drawing empat border, Pelacak Linear, Pelacak Petunjuk, Draw rekan satu tim (kebanyakaan digunakan untuk petak umpet), Draw nama item, Draw darah monster, Memantau jarak jauh seacara real time, Custom color, Custom ukuran, Custom filter, Perspektif pada semua sumber daya umum.</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-shituguanliqi"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Visual
+          </h4>
+        </a>
+        <p>Memodifikasi animasi Karakter, Memodifikasi ekspresi Karakter, Menampilkan item dari Peta Interaktif Teyvat, Buat browser Unity, Pengelihatan tak terbatas, Memodifikasi UID, Memodifikasi tampilan profile card, Custom cuaca, Membuka batasan limit FPS, Free camera, Memodifikasi Texture, Packet grabber, Kostumisai Private key.
```

Comment:
change it to `Memodifikasi animasi karakter, memodifikasi ekspresi karakter, menampilkan item dari Peta Interaktif Teyvat, buat browser Unity, Pengelihatan tak terbatas, memodifikasi UID, memodifikasi tampilan profile card, kustomisasi cuaca, membuka batasan FPS, free camera, memodifikasi tekstur, Packet Grabber, dan Kustomisasi Private Key.`